core/domain/role_domain.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
```

Comment:
I have changed the name. However, after changing name and running backend tests for role_services_test I am getting following error.

![image](https://cloud.githubusercontent.com/assets/9693472/26644631/074e66e4-4653-11e7-960a-cebd3a81424b.png)